Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2955,6 +2955,7 @@ void ProcessMolProps(RWMol *mol) {
           atom->setNumExplicitHs(ival - atom->getExplicitValence());
         }
       }
+      atom->clearProp(common_properties::molTotValence);
```

Comment:
this will leave the property intact if its value is zero (only hypothetical since we don't set the value if it's zero) or if there's a ZBO.
How about just moving this line down two lines so that the property is always cleared?

Reply:
Done!

Reply:
Great, thanks.
Will merge after the CI builds finish